.eslintrc.js

```diff
@@ -21,5 +18,6 @@ module.exports = {
     '@typescript-eslint/explicit-function-return-type': 'off',
     '@typescript-eslint/explicit-module-boundary-types': 'off',
     '@typescript-eslint/no-explicit-any': 'off',
+    '@typescript-eslint/no-unused-vars': 'error',
```

Comment:
Can we remove this one in favour of https://github.com/5afe/safe-client-gateway-nest/pull/50?

I think it's more clear if it's a separate PR and since it's already open

Reply:
I merged #50 here to see the effect when setting this config. Now it's #50 merged in main so I merged main here as well. 🙂 I usually don't proceed this way but this was trivial, sorry about the misunderstandings.